tests/Integration/NullValueTest.php

```diff
@@ -29,4 +29,21 @@ public function testNullAsValueForLink()
 
         $this->assertNull($entry->getLogo());
     }
+
+    /**
+     * @see https://github.com/contentful/contentful.php/issues/103
+     * @see https://github.com/contentful/contentful.php/pull/122
+     */
+    public function testFieldIsPresentButEmpty()
+    {
+        $client = new Client('irrelevant', 'rue07lqzt1co');
+
+        $client->parseJson($this->getFixtureContent('space.json'));
+        $client->parseJson($this->getFixtureContent('environment.json'));
+        $client->parseJson($this->getFixtureContent('content_type.json'));
+        $entry = $client->parseJson($this->getFixtureContent('entry_field_without_values.json'));
+
+        $this->assertNull($entry->get('logo'));
+        $this->assertSame([], $entry->get('phone'));
```

Comment:
Why is phone an array here?

Reply:
I just saw the Content Type definition, alles gut!

Reply:
Because in the content type it's defined as an array. The meaningfulness of the whole thing is not important, I just needed one field defined as array, and another defined as anything else 😉